lib/js/components/Pagination/Pagination.vue

```diff
@@ -0,0 +1,396 @@
+<template>
+	<div
+		class="ds-pagination"
+		:class="{
+			'-forceCompact': forceCompact,
+			'-centered': isCentered && !$slots.accessory,
+		}"
+	>
+		<div class="ds-pagination__itemsWrapper">
+			<div
+				v-if="navigationItems.length > 1 && !forceCompact"
+				class="ds-pagination__items -default"
+			>
+				<template v-for="(navigationItem, index) in navigationItems">
+					<div
+						v-if="isPage(navigationItem)"
+						:key="index"
+						class="ds-pagination__itemWrapper"
+						:class="{ '-touchable': currentPageSanitized !== navigationItem }"
+						role="link"
+						@click.prevent.stop="changePage(navigationItem)"
```

Comment:
```suggestion
						@click="changePage(navigationItem)"
```
I think I should've remove it when migrating away from `a` element.